admin/sql/create_tables.sql

```diff
@@ -158,6 +158,14 @@ CREATE TABLE statistics.sitewide (
 
 ALTER TABLE statistics.sitewide ADD CONSTRAINT stats_range_uniq UNIQUE (stats_range);
 
+CREATE TABLE recommendation_feedback (
+    id                      SERIAL, -- PK
+    user_id                 INTEGER NOT NULL, -- FK to "user".id
+    recording_mbid          UUID NOT NULL,
+    rating                  feedback_type_enum NOT NULL,
```

Comment:
wondering if we should have an index on this column ("get me all 'love' feedback for a user")